### PR TITLE
Update UltraPermissions_English.lang

### PR DESCRIPTION
**Pull request by tedztar**
Added new phrases

### CODE REVIEW

File: Translations/native/UltraPermissions_English.lang [hunk 1]
```diff
@@ -116,6 +126,30 @@ GUI_PERMEDITOR_NORMAL_RIGHTCLICK: **Right Click** to **select**
...
+GUI_PERMVIEW_EDITPERM_TITLE: Edit Permission
+GUI_PERMVIEW_EDITPERM_ACTION: Click to edit Permission
+GUI_PERMVIEW_EDITPERM_TYPEIN: Type in the new permission
+GUI_PERMVIEW_COPY_ACTION: Click to select a **target**
+GUI_PERMVIEW_NOMULTINAMEEDIT: you cannot rename multiple permissions at once
+GUI_STORAGESETTINGS_TITLE: Storage Settings
+GUI_STORAGESETTINGS_TYPEINWORLD: Type in a world name
+GUI_STORAGESETTINGS_NOWORLD_ACTION: Click to **restrict** this group to a specific World
+GUI_STORAGESETTINGS_WORLD_ACTION: Click to allow this group in all Worlds
+GUI_STORAGESETTINGS_NOSERVER_ACTION: Click to **restrict **this group to this Server
```
Comment (rlnt) [inline Translations/native/UltraPermissions_English.lang:149]: Is the character set intended?
**restrict **this

**Review by rlnt — COMMENTED** [review]
Explanation required

**Comment by tedztar** [comment]
Tech added phrases, didn’t add them to the GitHub.

**Comment by rlnt** [comment]
> Tech added phrases, didn’t add them to the GitHub.

I see that but I want to know whether the '*' characters are intended with a space in between like I mentioned in the review.

**Comment by tedztar** [comment]
I copied it all from what tech setup, so I presume yes

**Review by rlnt — APPROVED** [review]

**Pull request merged**